lib/awful/client.lua

```diff
@@ -599,6 +599,8 @@ function client.mark(c)
     end
 end
 
+
+
 --- Unmark a client and then call 'unmarked' hook.
```

Comment:
Adding empty lines in the middle of nowhere? (Well, between "mark" and "unmark", which even belong together, so this isn't some kind of separator)

(Yup, I decided to look through these commits)